src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="input input--large" data-qa="big">
+      <input
+        class="input__field input__field--large"
+        type="text"
+        name="big-search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="input input--small" data-qa="small">
+      <input
+        class="input__field input__field--small"
+        type="text"
+        name="small-search"
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
```

Comment:
Same as the previous comment, consider replacing `div` with a more semantic tag like `section` or `article`.